source/TimePicker.js

```diff
@@ -419,9 +414,31 @@
 					fmtParams.locale = this.locale;
 				}
 				var merFormatter = new ilib.DateFmt(fmtParams);
-				var am = ilib.Date.newInstance({hour:10});
-				var pm = ilib.Date.newInstance({hour:14});
-				this.meridiems = [merFormatter.format(am), merFormatter.format(pm)];
+				
+				this.meridiems = merFormatter.getMeridiemsRange(fmtParams);
```

Comment:
Hi @DavidUM, thanks for the updates! I think we can remove this assignment, right?